coralproject/talk
# [CORL-770] save comment sort order preference in local storage

## What does this PR do?
- stores user's comment sort order preference in localstorage and clears it on logout 

## How do I test this PR?
- choose comment sort order
- go to a new story or refresh the page
- sort order should be maintained

It occurs to me that we _could_ add some sort of flag to the local record management to tell it when to persist something to localstorage... but I suggest we do that if we come across this situation again.
